source/learning-wazuh/replace-stock-rule.rst

```diff
@@ -5,10 +5,10 @@
 Change the rules
 ================
 
-The `Wazuh Ruleset <https://github.com/wazuh/wazuh-ruleset>`_ is maintained by Wazuh, Inc.
+The `Wazuh Ruleset <https://github.com/wazuh/wazuh/ruleset>` is maintained by Wazuh, Inc.
 and is contributed to by the Wazuh community.  These stock rules are located in various files
-in ``/var/ossec/ruleset/rules/`` on the Wazuh manager and should not be edited in that location
-because they are overwritten when you upgrade Wazuh manager or perform a Wazuh Ruleset update.
+in ``/var/ossec/ruleset/rules/`` on the Wazuh Manager and should not be edited in that location
+because they are overwritten when you upgrade it.
```

Comment:
The `it` is not totally clear what is referring to, the manager, or the rules. I would say something like...

```
because they are overwritten when you make an upgrade.
```

Reply:
Updated!